### PR TITLE
feat: Modify Donation page.

### PR DESCRIPTION
**Pull request by Priyansh1825**
Multiple selection donation page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "@fortawesome/fontawesome-free": "^6.3.0",
+    "@fortawesome/fontawesome-svg-core": "^6.3.0",
+    "@fortawesome/free-brands-svg-icons": "^6.3.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
```
Comment (vaidehi27) [inline package.json:11]: why are we using emotion and styled components both? Also, what is the need to use fortawesome?

File: src/App.tsx [hunk 2]
```diff
@@ -21,7 +14,6 @@ export const App: React.FunctionComponent = () => {
         <Routes>
           <Route path="/" element={<Home />} />
           <Route path="/Pages/Home" element={<MyCarousel />} />
-          <Route path="/Pages/About" element={<AboutUs />} />
```
Comment (vaidehi27) [inline src/App.tsx:24]: why about us page is deleted?

File: package.json (hunk 1 above)
Comment (Priyansh1825) [reply package.json:11]: we did it by mistake.

File: src/App.tsx (hunk 2 above)
Comment (Priyansh1825) [reply src/App.tsx:24]: we link About us page in home page that why we deleted .

**Review by vaidehi27 — COMMENTED** [review]
Please create a PR where package `Pages` is replaced with `pages` ad `Components` is replaced with `components`. Unless the package has an immediate tsx file, the package name should follow camel case and not pascal case

**Pull request closed without merging**